environment.js

```diff
@@ -0,0 +1,26 @@
+var path = require('path');
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+var glob = require('glob');
+
+export.ensureDirectory = function(directoryName) {
+  if (!fs.existsSync(directoryName)) {
+    mkdirp.sync(directoryName);
+  }
+};
+
+export.loadSpecifications = function(mocha, directory) {
+   function addSpecs(error, files) {
```

Comment:
Any reason this isn't an anonymous function within the glob? I find this slightly confusing to read.